src/frameworks/angular/utils.ts

```diff
@@ -66,55 +66,32 @@ async function localesForTarget(
   return { locales, defaultLocale };
 }
 
-const enum ExpectedBuilder {
-  APPLICATION = "@angular-devkit/build-angular:application",
-  BROWSER_ESBUILD = "@angular-devkit/build-angular:browser-esbuild",
-  DEPLOY = "@angular/fire:deploy",
-  DEV_SERVER = "@angular-devkit/build-angular:dev-server",
-  LEGACY_BROWSER = "@angular-devkit/build-angular:browser",
-  LEGACY_NGUNIVERSAL_PRERENDER = "@nguniversal/builders:prerender",
-  LEGACY_DEVKIT_PRERENDER = "@angular-devkit/build-angular:prerender",
-  LEGACY_SERVER = "@angular-devkit/build-angular:server",
-  LEGACY_NGUNIVERSAL_SSR_DEV_SERVER = "@nguniversal/builders:ssr-dev-server",
-  LEGACY_DEVKIT_SSR_DEV_SERVER = "@angular-devkit/build-angular:ssr-dev-server",
+const enum BuilderType {
+  DEPLOY = "deploy",
+  DEV_SERVER = "dev-server",
+  SSR_DEV_SERVER = "ssr-dev-server",
+  SERVER = "server",
+  BROWSER = "browser",
+  BROWSER_ESBUILD = "browser-esbuild",
+  APPLICATION = "application",
+  PRERENDER = "prerender",
 }
 
-const enum ThirdPartyBuilder {
-  CUSTOM_WEBPACK_DEV_SERVER = "@angular-builders/custom-webpack:dev-server",
-  CUSTOM_WEBPACK_SERVER = "@angular-builders/custom-webpack:server",
-  CUSTOM_WEBPACK_BROWSER = "@angular-builders/custom-webpack:browser",
-  NGX_ENV_APPLICATION = "@ngx-env/builder:application",
-  NGX_ENV_DEV_SERVER = "@ngx-env/builder:dev-server",
-}
-
-const DEV_SERVER_TARGETS: string[] = [
-  ExpectedBuilder.DEV_SERVER,
-  ExpectedBuilder.LEGACY_NGUNIVERSAL_SSR_DEV_SERVER,
-  ExpectedBuilder.LEGACY_DEVKIT_SSR_DEV_SERVER,
-  ThirdPartyBuilder.CUSTOM_WEBPACK_DEV_SERVER,
-  ThirdPartyBuilder.CUSTOM_WEBPACK_SERVER,
-  ThirdPartyBuilder.NGX_ENV_DEV_SERVER,
-];
+const DEV_SERVER_TARGETS: string[] = [BuilderType.DEV_SERVER, BuilderType.SSR_DEV_SERVER];
 
-function getValidBuilders(purpose: BUILD_TARGET_PURPOSE): string[] {
+function getValidBuilderTypes(purpose: BUILD_TARGET_PURPOSE): string[] {
```

Comment:
we should improve the types here
```suggestion
function getValidBuilderTypes(purpose: BUILD_TARGET_PURPOSE): BuilderType[] {
```

Reply:
done 🫡